src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { setupUi } from './tests/camera-mode/ui'
-import './tests/camera-mode/index.test'
+import './tests/transform/index.test'
+// import './tests/camera-mode/index.test'
```

Comment:
```suggestion
import './tests/camera-mode/index.test'
```
check this before merge